modules/gapi/include/opencv2/gapi/oak/oak.hpp

```diff
@@ -89,28 +89,69 @@ G_API_OP(GSobelXY, <GFrame(GFrame, const cv::Mat&, const cv::Mat&)>, "org.opencv
     }
 };
 
+G_API_OP(GCopy, <GFrame(GFrame)>, "org.opencv.oak.copy") {
+    static GFrameDesc outMeta(const GFrameDesc& in) {
+        return in;
+    }
+};
+
+// FIXME: add documentation on operations below
+
 GAPI_EXPORTS GArray<uint8_t> encode(const GFrame& in, const EncoderConfig&);
 
 GAPI_EXPORTS GFrame sobelXY(const GFrame& in,
                             const cv::Mat& hk,
                             const cv::Mat& vk);
 
+GAPI_EXPORTS GFrame copy(const GFrame& in);
+
 // OAK backend & kernels ////////////////////////////////////////////////////////
 GAPI_EXPORTS cv::gapi::GBackend backend();
 GAPI_EXPORTS cv::gapi::GKernelPackage kernels();
 
 // Camera object ///////////////////////////////////////////////////////////////
 
-struct GAPI_EXPORTS ColorCameraParams {};
+struct GAPI_EXPORTS ColorCameraParams {
+    /**
+     * Camera output specifies what type of data to pull from OAK camera on device.
+     * raw - RAW10 bayer data
+     * isp - YUV420 planar
+     * still - NV12
+     * preview - RGB (or BGR planar/interleaved if configured)
+     * video - NV12
+     */
+    enum class CameraOutput: int { VIDEO, PREVIEW, STILL, ISP, RAW };
+
+    // Video is suitable for encoder and some of the CV operations
+    // Note: for inference it's recommended to use PREVIEW
+    CameraOutput camera_output = CameraOutput::VIDEO;
```

Comment:
Can't find where it's used

Reply:
Right, it's been decided that it will be handled by the backend (e.g. `preview` will be linked to infer, `video` to cv, etc). Removed